### PR TITLE
Fix some timing and definitions

### PR DESCRIPTION
**Pull request by dieris**
Fix units and try to better approximate the set rate

### CODE REVIEW

File: common/+deviceDrivers/YokoGS200.m [hunk 1]
```diff
@@ -68,13 +68,13 @@
...
-            step = rate*1e3; %1 ms step
+            step = rate*1e-3; %1 ms step
+            nsteps = round(abs(DeltaV/step));
             for j=1:nsteps
                 CurrentV=CurrentV+j*step;
                 obj.write([':SOURCE:LEVEL ' num2str(CurrentV)]);
-                pause(1e-3)
+                pause(0.6e-3) %each set instruction takes ~0.4 ms
             end
-            obj.write([':SOURCE:LEVEL ' num2str(Vset)]);
```
Comment (caryan) [inline common/+deviceDrivers/YokoGS200.m]: Why did you remove this? It seems you might not end up at Vset. Say `CurrentV = 0` is 0 and `Vset = 3`  then `DeltaV=3` and if `step = 2` then you'll get two steps and end up at 4.

**Comment by caryan** [comment]
@dieris what do you think about a74488a

**Comment by dieris** [comment]
looks good
